google/cloud/spanner_dbapi/client_side_statement_executor.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 20203 Google LLC All rights reserved.
```

Comment:
super-nit: (here and elsewhere) I think it is a bit optimistic to think that any of us will be alive at this copyright year :-)

Reply:
Haha. Reduced it by 18180 years :)